gallery/transforms/plot_transforms_illustrations.py

```diff
@@ -4,56 +4,34 @@
 ==========================
 
 .. note::
-    Try on `collab <https://colab.research.google.com/github/pytorch/vision/blob/gh-pages/main/_generated_ipynb_notebooks/plot_transforms.ipynb>`_
-    or :ref:`go to the end <sphx_glr_download_auto_examples_others_plot_transforms.py>` to download the full example code.
+    Try on `collab <https://colab.research.google.com/github/pytorch/vision/blob/gh-pages/main/_generated_ipynb_notebooks/plot_transforms_illustrations.ipynb>`_
+    or :ref:`go to the end <sphx_glr_download_auto_examples_transforms_plot_transforms_illustrations.py>` to download the full example code.
 
 This example illustrates the various transforms available in :ref:`the
 torchvision.transforms module <transforms>`.
 """
 
 # sphinx_gallery_thumbnail_path = "../../gallery/assets/transforms_thumbnail.png"
 
+# %%
```

Comment:
This introduces an empty box:

![image](https://github.com/pytorch/vision/assets/6849766/81237c39-48e1-40d9-a266-f096f853f906)